conda/activate.py

```diff
@@ -13,10 +13,11 @@
 import sys
 from textwrap import dedent
 
+from cytoolz.itertoolz import concatv, drop
```

Comment:
There is a `tlz` namespace that tries `cytoolz` first and falls back to `toolz`. It also works regardless of which is installed. This is what we do in Dask ( https://github.com/dask/dask/pull/5949#issuecomment-595407707 )

```suggestion
from tlz.itertoolz import concatv, drop
```

Reply:
hm interesting, and I see that `cytoolz` lists `toolz` as a dependency so it'll always be present

Reply:
Thanks for the suggestion. I just updated all of our import statements to use `tlz` instead of `cytoolz` 👍

Reply:
Thanks Travis! 🙏